src/connectors/autocomplete/connectAutocomplete.js

```diff
@@ -0,0 +1,119 @@
+import escapeHits, { tagConfig } from '../../lib/escape-highlight';
+import { checkRendering } from '../../lib/utils';
+
+const usage = `Usage:
+var customAutcomplete = connectAutocomplete(function render(params, isFirstRendering) {
+  // params = {
+  //   indices,
+  //   hits,
+  //   refine,
+  //   currentRefinement
+  // }
+});
+search.addWiget(customAutcomplete({
+  [ indices ],
```

Comment:
what about also returning `index` (first index of `indices`) for use cases when you know there will only be one index? Seems to be a way how you can have both the simplicity of a single index when you need it, but avoid the two separate return paths RIS has, cc @samouss

Reply:
We don't need to handle the single index in a special case. When it's an array both cases are treated the same. You can still loop on array of one item.